src/styles/blocks/cardMain.scss

```diff
@@ -0,0 +1,85 @@
+.card {
```

Comment:
rename file just to `card.scss` to match the block name

Reply:
not fixed 
![image](https://github.com/mate-academy/layout_product-cards/assets/70009205/773edcce-3465-45f3-bfd5-11a278c4b14d)